### PR TITLE
Update compiler download instructions to point to ARM Developer, not CodeSourcery.

### PR DESCRIPTION
**Pull request by szabodabo**
Documenting my experience building openmrn for the first time.

### CODE REVIEW

File: doc/getting_started.dox [hunk 1]
```diff
@@ -57,30 +57,39 @@
...
- * The OpenMRN build system will look for the compiler at /opt/CodeSourcery or
- * ~/CodeSourcery.  The compiler includes a copy of Newlib used by OpenMRN.
- *
- * The file inside the download should be named "arm-2013.05-23-arm-none-eabi-i686-pc-linux-gnu.tar". 
- * More variants of the compiler are available, be sure to select this one.
- * Unpacking this file gives a directory named arm-2013.05
+
+ *  The compiler used is GCC as distributed with ARM's GNU Embedded Toolchain
+ * (as of this writing, we're using version `6-2017-q1-update`). This compiler
+ * may be [downloaded the ARM Developer site](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads).
```
Comment (balazsracz) [inline doc/getting_started.dox]: downloaded from

File: doc/getting_started.dox [hunk 2]
```diff
@@ -57,30 +57,39 @@
...
+ * The OpenMRN build system will look for the compiler at `/opt/armgcc` (for
```
Comment (balazsracz) [inline doc/getting_started.dox]: /opt/armgcc/default

File: doc/getting_started.dox (hunk 1 above)
Comment (szabodabo) [reply doc/getting_started.dox]: done

File: doc/getting_started.dox (hunk 2 above)
Comment (szabodabo) [reply doc/getting_started.dox]: done

**Comment by szabodabo** [comment]
thanks for the review, pushed a tiny fix commit.

**Pull request merged**